### ISSUE TITLE
Testing code error

### ISSUE THREAD

**Issue by praful1993** [MIVRC/SeaNet-PyTorch#4]
How to solve this error??
(pytorch0.4) G:\EDGE\seanet-pytorch-master\Test\code>python main.py --data_test MyImage --scale 4 --model SEAN --pre_train F:/Nancy/seanet-pytorch-master/Test/model/SEAN_x4.pt --test_only --chop --save "SEAN" --testpath ../LR/LRBI --testset Set5
usage: main.py [-h] [--debug] [--template TEMPLATE] [--n_threads N_THREADS]
               [--cpu] [--n_GPUs N_GPUS] [--seed SEED] [--dir_data DIR_DATA]
               [--dir_demo DIR_DEMO] [--data_train DATA_TRAIN]
               [--data_test DATA_TEST] [--benchmark_noise] [--n_train N_TRAIN]
               [--n_val N_VAL] [--offset_val OFFSET_VAL] [--ext EXT]
               [--scale SCALE] [--patch_size PATCH_SIZE]
               [--rgb_range RGB_RANGE] [--n_colors N_COLORS] [--noise NOISE]
               [--chop] [--model MODEL] [--act ACT] [--pre_train PRE_TRAIN]
               [--extend EXTEND] [--n_resblocks N_RESBLOCKS]
               [--n_feats N_FEATS] [--res_scale RES_SCALE]
               [--shift_mean SHIFT_MEAN] [--precision {single,half}] [--reset]
               [--test_every TEST_EVERY] [--epochs EPOCHS]
               [--batch_size BATCH_SIZE] [--split_batch SPLIT_BATCH]
               [--self_ensemble] [--test_only] [--gan_k GAN_K] [--lr LR]
               [--lr_decay LR_DECAY] [--decay_type DECAY_TYPE] [--gamma GAMMA]
               [--optimizer {SGD,ADAM,RMSprop}] [--momentum MOMENTUM]
               [--beta1 BETA1] [--beta2 BETA2] [--epsilon EPSILON]
               [--weight_decay WEIGHT_DECAY] [--loss LOSS]
               [--skip_threshold SKIP_THRESHOLD] [--n_resgroups N_RESGROUPS]
               [--reduction REDUCTION] [--testpath TESTPATH]
               [--testset TESTSET] [--degradation DEGRADATION]
main.py: error: unrecognized arguments: --save SEAN

**Comment by zkh123456789** [comment]
I also encountered this problem, have you solved it?

**Comment by IceyWuu** [comment]
Gosh, code in this repo is a mess! You can add parser.add_argument('--save', type=str, default='test', help='file name to save') in the test option.py file, this option file miss so many args, can find in the train option.py

> I also encountered this problem, have you solved it?
